development.ini

```diff
@@ -5,10 +5,7 @@ pyramid.includes =
     pyramid_debugtoolbar
     pyramid_tm
 
-SQLITE_DATABASE_URL = sqlite:///%(here)s/sqltutorial.sqlite
-POSTGRES_DATABASE_URL = postgresql+psycopg2://postgres:localpassword@localhost:5432/noderegistry
-sqlalchemy.url = %(POSTGRES_DATABASE_URL)s
-
+sqlalchemy.url = sqlite:///%(here)s/network_manager.sqlite
```

Comment:
As Francis mentioned in his comment, we don't need to specify alternative urls in this file, we can just replace the file with any updated settings if needed. Let's keep this simple and stick with the sqlite url as a default.